### ISSUE TITLE
JSONL format:

### ISSUE THREAD

**Issue by al-swaiti** [voicepowered-ai/VibeVoice-finetuning#12]
I attempted training several times but the results were very poor (i tested it using comfyui)
https://huggingface.co/ABDALLALSWAITI/vibevoice-arabic-v1,
the data i used https://huggingface.co/datasets/mozilla-foundation/common_voice_17_0/viewer/ar?views%5B%5D=ar_train .
i generated 
this jsonl 
{"text": "Speaker 0: وما كان ربك ليهلك القرى بظلم وأهلها مصلحون", "audio": "vibevoice_data/train_audio/audio_000000.wav"}
{"text": "Speaker 0: أقدر تعاونك معنا.", "audio": "vibevoice_data/train_audio/audio_000001.wav"}
{"text": "Speaker 0: وَلُوطًا إِذْ قَالَ لِقَوْمِهِ أَتَأْتُونَ الْفَاحِشَةَ مَا سَبَقَكُمْ بِهَا مِنْ أَحَدٍ مِنَ الْعَالَمِينَ", "audio": "vibevoice_data/train_audio/audio_000002.wav"}
{"text": "Speaker 0: فلما رأوا بأسنا قالوا آمنا بالله وحده وكفرنا بما كنا به مشركين", "audio": "vibevoice_data/train_audio/audio_000003.wav"}
{"text": "Speaker 0: لمن هذا الكتاب؟", "audio": "vibevoice_data/train_audio/audio_000004.wav"}
{"text": "Speaker 0: إنه خطير للغاية.", "audio": "vibevoice_data/train_audio/audio_000005.wav"}
{"text": "Speaker 0: بل متعت هؤلاء وآباءهم حتى جاءهم الحق ورسول مبين", "audio": "vibevoice_data/train_audio/audio_000006.wav"}
{"text": "Speaker 0: فَيَصِيرُ بِالرَّمْزِ سَائِرًا وَفِي الصُّحُفِ مُخَلَّدًا", "audio": "vibevoice_data/train_audio/audio_000007.wav"}
{"text": "Speaker 0: أنا شريكك.", "audio": "vibevoice_data/train_audio/audio_000008.wav"}
............................... etc
what i missed !

**Comment by jpgallegoar-vpai** [comment]
Impossible to know. You give no info about your training parameters, no info about loss graphs... Besides common_voice is notoriously bad for fine-tuning

**Comment by al-swaiti** [comment]
i just want to know if my jsonl correct for training , any good dataset , to follow for any language so ican compare?walkthrough !?
 , thanks ,
(i found his results good https://huggingface.co/Cseti/VibeVoice_7B_Diffusion-head-LoRA_Hungarian-CV17 )

**Comment by al-swaiti** [comment]
https://github.com/vibevoice-community/VibeVoice/blob/main/FINETUNING.md
thnx
